pkg/materialize/role_parameter.go

```diff
@@ -0,0 +1,44 @@
+package materialize
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+)
+
+type RoleParameterBuilder struct {
+	ddl           Builder
+	roleName      string
+	variableName  string
+	variableValue string
+}
+
+func NewRoleParameterBuilder(conn *sqlx.DB, roleName, variableName, variableValue string) *RoleParameterBuilder {
+	return &RoleParameterBuilder{
+		ddl:           Builder{conn, System},
+		roleName:      roleName,
+		variableName:  variableName,
+		variableValue: variableValue,
+	}
+}
+
+func (b *RoleParameterBuilder) Set() error {
+	q := fmt.Sprintf(`ALTER ROLE "%s" SET "%s" TO '%s';`, b.roleName, b.variableName, b.variableValue)
```

Comment:
Should we be quoting these?

Reply:
Yep, I think so, do you believe that the quoting here is unnecessary? 

The idea here is that in case that a role has a special character like a space, it will not work without the quotes:

```sql
ALTER ROLE "some role" SET "cluster" = 'some cluster';
```

Reply:
Oh, I was wondering about using the `QuoteString` function instead.

Reply:
Ah yes 🤦! Good catch, thanks Aru! 🙌 

Just updated this, let me know what you think.